assets/css/lesson-editor.scss

```diff
@@ -0,0 +1,34 @@
+@import '~@wordpress/base-styles/breakpoints';
+
+#content-drip-upgrade {
+	.inside {
+		display: flex;
+		flex-direction: column;
+
+		@media (min-width: $break-medium)  {
+			flex-direction: row;
+			justify-content: space-between;
+			gap: 1em;
+		}
+	}
+
+	.sensei-content-drip-upgrade {
+		&__descriptions {
+			max-width: 410px;
+
+			a:after {
```

Comment:
Hey you can add  ```vertical-align: text-top; ``` to align the icon with text

Reply:
TIL, thank you! Fixed in 99099dd2faebf2e124d510e5d6986941328b1b26.